js/packages/screenshoter/test/e2e/web/region-fully.spec.js

```diff
@@ -36,4 +36,17 @@ describe('screenshoter web', () => {
       logger,
     })
   })
+
+  it('take full region screenshot with "css" scrolling', async () => {
```

Comment:
I don't like that this test is called exactly the same as the one above. I recommend
```suggestion
  it('take full region screenshot after manual scroll with "css" scrolling', async () => {
```